blueocean-web/src/main/js/try.js

```diff
@@ -1,12 +1,7 @@
 var $ = require('jquery-detached').getJQuery();
 
 function getAppUrl() {
-    const rootUrl = $('head').attr('data-rooturl');
-    if (!rootUrl) {
-        return '';
-    } else {
-        return rootUrl;
-    }
+    return rootUrl = $('head').attr('data-rooturl');
```

Comment:
So I don't see how this fixes anything. Correct me if I'm wrong, but all I see here is that you removed the checks on the `rootUrl` value got from the `data-rooturl` attribute on the `<head>` element. How is that fixing the problem described in [JENKINS-41408](https://issues.jenkins-ci.org/browse/JENKINS-41408) ?

As a matter of interest; when you run without this change, what's the value of `head@data-rooturl` when you inspect the source?